crates/ty_python_semantic/resources/mdtest/pep695_type_aliases.md

```diff
@@ -120,6 +120,36 @@ def f(x: IntOrStr, y: str | bytes):
     reveal_type(z)  # revealed: (int & ~AlwaysFalsy) | str | bytes
 ```
 
+## Multiple layers of union aliases
+
+```py
+class A:
+    pass
+
+class B:
+    pass
+
+class C:
+    pass
+
+class D:
+    pass
+
+class E:
+    pass
+
+class G[T]:
+    pass
```

Comment:
nit: Black formats these more concisely if we use `...`, since we pass `--pyi` to blacken-docs

```suggestion
class A: ...
class B: ...
class C: ...
class D: ...
class E: ...
class G[T]: ...
```